src/legacy/core_plugins/console/np_ready/public/application/components/console_menu.tsx

```diff
@@ -91,6 +104,26 @@ export class ConsoleMenu extends Component<Props, State> {
     this.props.autoIndent(event);
   };
 
+  addCopyToast = () => {
+    const toast: Toast = {
+      id: String(this.state.toastId),
+      title: i18n.translate('console.consoleMenu.copyRequestAsCurlMessage', {
+        defaultMessage: 'Request copied as cURL',
+      }),
+      color: 'success',
```

Comment:
I would consider this kind of feedback to be neutral, so I think we should leave out the `color`. I think of "success" as more appropriate for successful completion of an async action.